<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-I41HQ3G" failures="1" tests="5" name="Personal Loan" time="0.049" errors="0" timestamp="2024-08-22T21:40:29 SAST">
  <testcase classname="test.day1" name="Demo" time="0.01"/>
  <testcase classname="test.day2" name="ploan" time="0.001"/>
  <testcase classname="test.day4" name="LoginHomeAPI" time="0.002"/>
  <testcase classname="test.day4" name="mobileHomeLogin" time="0.002"/>
  <testcase classname="test.day4" name="webHomelogin" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day4.webHomelogin(day4.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- webHomelogin -->
  <testcase classname="test.day1" name="Demo2" time="0.0">
    <ignored/>
  </testcase> <!-- Demo2 -->
</testsuite> <!-- Personal Loan -->
